src/App.tsx

```diff
@@ -1,61 +1,179 @@
 import './App.scss';
+import {
+  FormControl,
+  Grid,
+  InputLabel,
+  MenuItem,
+  Paper,
+  Select,
+  SelectChangeEvent,
+  TextField,
+} from '@mui/material';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { useState } from 'react';
+import Button from '@mui/material/Button';
+import { TodoList } from './components/TodoList/TodoList';
+
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { User } from './types/User';
+import { Task } from './types/Task';
+
+const tasksFromServer = todosFromServer.map(todo => {
+  return {
+    ...todo,
+    user: { ...usersFromServer.find(user => user.id === todo.userId) },
```

Comment:
We don't need to create a copy for found user

Reply:
Fix